package.json

```diff
@@ -41,7 +41,7 @@
     "jest": "^26.4.2",
     "mocha": "^8.1.3",
     "release-it": "^14.4.1",
-    "saucectl": "^0.31.2",
+    "saucectl": "^0.33.1",
```

Comment:
To ensure having the latest version of `saucectl`, you can use the `saucectl-run-action` to install it:
Like here: 
https://github.com/saucelabs/sauce-testcafe-runner/blob/master/.github/workflows/test.yml#L91

Reply:
I think pinning is better in this case. Moving from 0.31 to 0.33 introduced a "breaking" change to our tests (`--test-env` defaults to `sauce` now instead of `docker`). "Breaking" there in quotes because it didn't end up failing jobs, but it did change the intended behaviour of the tests.

Reply:
IMO there is pro & cons to both solution. Pinning prevent from having those kind of changes issues due to defaults changes. Having the latest one ensures this is still working as expected with the latest one.
I'm ok with both solutions :)